### ISSUE TITLE
Configurable persistence backend

### ISSUE THREAD

**Issue by lesnitsky** [rodydavis/settings_manager#3]
I want to be able to configure persistence backend and use other storages, not just `shared_preferences`

**Comment by rodydavis** [comment]
Yep I am going to add firestore, SQLite and custom solutions too! I’ll update soon 😉
